Marlin/src/pins/pins_MKS_SBASE.h

```diff
@@ -129,7 +129,6 @@
 #if ENABLED(MK2_MULTIPLEXER)
   #define E_MUX0_PIN         P0_17   // J7-4
   #define E_MUX1_PIN         P0_16   // J7-5
-  #define E_MUX2_PIN         P0_15   // J7-6
```

Comment:
Are you able to find an alternative pin for the multiplexer MUX2?

Reply:
P1_23, 2_12, P2_11 and P4.28 are free at connector J8.

But the reason i removed it is because it's not used on the Prusa multiplexer, will only be needed if they design a new board for 8 ? extruders....

Reply:
If you think we should include MUX2 we could move all of them to J8, cleaner wiring...

Reply:
Ok if i ask a question here @thinkyhead , find you guy's email i not real easy, sometimes there are questions that just dont fit into a github and are of noe interest to other's.

I have a few questions and would like to send a email to you , @tcm0116 , @Roxy-3D , how can we fix that ? 😄

Reply:
I'm a private person, so don't like to share my email too much. But, if you write to <anything> @thinkyhead.com it comes to me.

Reply:
> it's not used on the Prusa "like" MM multiplexer

True, but their code includes it, so I expect there's room for future expansion.

Reply:
ok, should i move them all to J8 ?

Reply:
Done :)